partials/new_device_login.html

```diff
@@ -1,4 +1,8 @@
 <% extends "partials/base/base_text_only.html" %>
 
 <% block heading %>{{t "New sign-in to Firefox"}}<% endblock %>
-<% block text_primary %>{{{ device }}}<br/>{{ timestamp }}<% endblock %>
+<% block text_primary %>{{{ device }}}<br/>
+<span style="display:block;">IP address: {{ ip }}</span>
```

Comment:
Yes, and me may also want to put the ":" in there for translation as well.  See for example `{{t "Alternatively:"}}` in other templates.